src/main/config.ts

```diff
@@ -21,7 +21,9 @@ export const defaultConfiguration = {
   radioEffects: 'on' as RadioEffects,
   hardwareType: 0,
   radioGain: 0,
-  alwaysOnTop: 'never' as AlwaysOnTopMode
+  alwaysOnTop: 'never' as AlwaysOnTopMode,
+  showExpandedRx: false,
+  transparentMiniMode: false
 };
```

Comment:
Since this is adding new properties to the config I suggest bumping the config version to 3.

That will also require a small change to the `V1ToV2` upgrade method which I now realize has a logic bug in it. This ine:

https://github.com/pierr3/TrackAudio/blob/5ba0441034a096153b66a42aef95cde003a1b9df/src/main/config.ts#L137

Should be `if (!config.version || config.version < 2)` and wrap the entire upgrade code instead of returning.

Reply:
Yep this sounds good to me

Reply:
Like this? https://github.com/pierr3/TrackAudio/blob/1baa6768d343229cababaf59f4435bc501b2294f/src/main/config.ts#L137-L166

Sorry whilst I'm writing this message my brain feels disconnected from my body so just make sure this looks right 😆

Reply:
I think so. I pulled the changes locally and am going to look to make sure. It's been a while since I added the migration stuff.

Reply:
I do believe this is correct. TBH the test for < 2 probably isn't needed since the original config files getting upgraded from this code never had a version property so the `!config.version` is sufficient, but it does make it clear what's happening.